rules/docker-telemetry.mk

```diff
@@ -29,6 +29,10 @@ endif
 
 $(DOCKER_TELEMETRY)_CONTAINER_NAME = telemetry
 $(DOCKER_TELEMETRY)_RUN_OPT += --privileged -t
+$(DOCKER_TELEMETRY)_RUN_OPT += --memory 400m
+$(DOCKER_TELEMETRY)_RUN_OPT += --memory-swap 400m
+$(DOCKER_TELEMETRY)_RUN_OPT += --cpus 0.3
+$(DOCKER_TELEMETRY)_RUN_OPT += --oom-kill-disable
```

Comment:
>oom [](http://example.com/codeflow?start=33&length=3)

Offline discussed. will update this PR and will also update the Monit script memory_checkerto read the memory usage of telemetry from file instead of docker stats.